src/js/KeyCodes.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by cmeyers on 11/4/16.
+ */
+
+export default {
+    TAB: 9,
+    ENTER: 13,
+    ESC: 27,
+    SPACEBAR: 32,
+    PAGE_UP: 33,
+    PAGE_DOWN: 34,
+    END: 35,
+    HOME: 36,
+    ARROW_LEFT: 37,
+    ARROW_UP: 38,
+    ARROW_RIGHT: 39,
+    ARROW_DOWN: 40,
+};
```

Comment:
We should probably double-check how page up/down and home/end work across keyboard layouts and OSs to make sure it behaves as we expect. I've been reading a lot recently about others' problems with it.